.travis.yml

```diff
@@ -6,7 +6,7 @@ language: node_js
 node_js:
   - "6"
 script:
-  - bash ./scripts/deploy.sh
+  - bash ./src/scripts/deploy.sh
```

Comment:
Ok I'm going to leave this for now but may revert this change in a follow up PR. It's nothing major but I think it's good to separate build process related scripts from actual the actual javascript that runs the site (though the line is somewhat blurry for static site generators).